Add build target: debian bookworm

piotrpawlaczek opened the pull request:
Add build target debian bookworm. 

The official wkhtmltopdf package does not have the QT patch and Debian bookworm RC is currently in "hard freeze" mode, as stated in this announcement: https://lists.debian.org/debian-devel-announce/2023/03/msg00004.html. 

The arm64 architecture build on my machine is successful.

```bash
{:timestamp=>"2023-04-10T12:17:43.124788+0000", :message=>"epoch in Version is set", :epoch=>"1", :level=>:warn}
{:timestamp=>"2023-04-10T12:18:48.458443+0000", :message=>"Created package", :path=>"wkhtmltox_0.12.6.1-1.bookworm_arm64.deb"}
```

piotrpawlaczek committed:
Add build target: debian bookworm

ashkulz commented on build.yml:
```diff
@@ -60,6 +60,30 @@ docker-targets:
+      libfreetype6
+      libjpeg62-turbo
+      libpng16-16
+      libssl1.1
```
This looks wrong, as [`libssl-dev` depends on libssl3](https://packages.debian.org/bookworm/libssl-dev).

ashkulz commented on .azure-pipelines.yml:
```diff
@@ -16,6 +16,8 @@ jobs:
   strategy:
     matrix:
       # 64-bit x86
+      bookworm_amd64:
+        target: bookworm-amd64
```
Please revert these changes, as Azure Pipelines is no longer used.

ashkulz commented on build.yml:
```diff
@@ -60,6 +60,30 @@ docker-targets:
+      from: debian:bookworm
+      python: python3
+    output: deb
+    matrix: ['amd64', 'arm64']
```
Can you keep it the same as `bullseye` i.e
```suggestion
    matrix: ['amd64', 'i386', 'arm64', 'armhf', 'armel', 'ppc64el']
```
assuming that docker images are available for all the above architectures?

ashkulz reviewed (changes requested):
Also, I'd appreciate if you would create a configuration for Raspberry Pi (`raspberrypi.bookworm-armhf`) -- you can probably copy/paste the configuration with a few changes, it'd done similarly for `bullseye` :+1:

piotrpawlaczek committed:
Address feedback

- revert azure pipline changes
- add support for 'armhf', 'armel', 'ppc64el', `i386` architecures
- add configuration for raspberrypi.bookworm-armhf
- upgrde to libssl3 dep

piotrpawlaczek commented:
@ashkulz Thx, would it be possible to publish the bookworm .deb packages here?

ashkulz commented:
Like [I said on the ticket](https://github.com/wkhtmltopdf/packaging/issues/149#issuecomment-1501713022), it doesn't make sense to publish packages as there's no official release yet.

ashkulz commented:
@piotrpawlaczek published a [new release 0.12.6.1-3](https://github.com/wkhtmltopdf/packaging/releases/tag/0.12.6.1-3), some builds are still ongoing due to hitting the 6-hour cutoff.
